src/crsAdapter/MerlinAdapter.js

```diff
@@ -671,11 +673,12 @@ class MerlinAdapter {
         }
 
         adapterService.travellers.forEach((serviceTraveller) => {
+            const normalizedTraveller = this.helper.traveller.normalizeTraveller(serviceTraveller);
             let travellerIndex = this.getNextEmptyTravellerIndex(crsData);
             let traveller = crsData.TravellerBlock.PersonBlock.PersonRow[travellerIndex];
 
             traveller.Salutation = CONFIG.crs.gender2SalutationMap[serviceTraveller.gender];
```

Comment:
then please use here all the `normalizedTraveller` values - like it is in the BewotecAdapter.

Reply:
done.